modules/custom/az_paragraphs/src/Plugin/paragraphs/Behavior/AZTextWithBackgroundParagraphBehavior.php

```diff
@@ -0,0 +1,108 @@
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\paragraphs\ParagraphInterface;
+
+/**
+ * Provides a behavior for text with background.
+ *
+ * @ParagraphsBehavior(
+ *   id = "az_text_background_paragraph_behavior",
+ *   label = @Translation("Quickstart Text with Background Paragraph Behavior"),
+ *   description = @Translation("Provides class selection for text with background."),
+ *   weight = 0
+ * )
+ */
+class AZTextWithBackgroundParagraphBehavior extends AZDefaultParagraphsBehavior {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildBehaviorForm(ParagraphInterface $paragraph, array &$form, FormStateInterface $form_state) {
+    $config = $this->getSettings($paragraph);
+
+    $form['text_background_title'] = [
+      '#title' => $this->t('Title'),
+      '#type' => 'textfield',
+      '#default_value' => $config['text_background_title'] ?? '',
+      '#description' => $this->t('Title of the Paragraph'),
+    ];
+
+    $form['text_background_title_centered'] = [
+      '#title' => $this->t('Centered'),
+      '#type' => 'checkbox',
+      '#default_value' => $config['text_background_title_centered'] ?? '',
+      '#description' => $this->t('Centers the title if checked.'),
+      '#return_value' => 'text-center',
+    ];
+
+    parent::buildBehaviorForm($paragraph, $form, $form_state);
+
+    $form['az_display_settings']['text_background_color'] = [
+      '#title' => $this->t('Background Color'),
+      '#type' => 'select',
+      '#options' => [
+        '' => $this->t('None'),
+        'bg-red' => $this->t('Arizona Red'),
+        'bg-blue' => $this->t('Arizona Blue'),
+        'bg-sky' => $this->t('Sky'),
+        'bg-oasis' => $this->t('Oasis'),
+        'bg-azurite' => $this->t('Azurite'),
+        'bg-midnight' => $this->t('Midnight'),
+        'bg-bloom' => $this->t('Bloom'),
+        'bg-chili' => $this->t('Chili'),
+        'bg-cool-gray' => $this->t('Cool Gray'),
+        'bg-warm-gray' => $this->t('Warm Gray'),
+        'bg-leaf' => $this->t('Leaf'),
+        'bg-river' => $this->t('River'),
+        'bg-silver' => $this->t('Silver'),
+        'bg-ash' => $this->t('Ash'),
+      ],
+      '#default_value' => $config['text_background_color'] ?? '',
+      '#description' => $this->t('<br><big><b>Important:</b></big> Site editors are responsible for accessibility and brand guideline considerations.<ul><li>To ensure proper color contrast, use the text color accessibility test at the bottom of the <a href="http://uadigital.arizona.edu/ua-bootstrap/colors.html" target="_blank">UA Bootstrap color documentation</a>.</li><li>For guidance on using the University of Arizona color palette, visit <a href="https://brand.arizona.edu/ua-color-palette" target="_blank">brand.arizona.edu</a>.</li></ul>'),
+    ];
+
+    $form['az_display_settings']['text_background_pattern'] = [
+      '#title' => $this->t('Background Pattern'),
+      '#type' => 'select',
+      '#options' => [
+        '' => $this->t('None'),
```

Comment:
I could be wrong, but I think the options should be:

```
        'bg-triangles-top-left' => $this->t('Triangles Left'),
        'bg-triangles-centered' => $this->t('Triangles Centered'),
        'bg-triangles-top-right' => $this->t('Triangles Right'),
        'bg-trilines' => $this->t('Trilines'),
```
Sorry for the runaround on this, and if the options above are implemented, I don't know that an additional css file is necessary.  So you can probably remove that library altogether, Maybe. haha

Reply:
the css still has the padding, so I guess I will leave it in.

Reply:
We all forgot that the class names changed between our bootstrap versions from `bg-triangles-left` to `bg-triangles-top-left` && `bg-triangles-right` to `bg-triangles-top-right`

This additional library with the old class names could still be useful for migration purposes though. I will have to think on it.

Reply:
I'll keep the CSS file for the padding style, but if the new classes work, I can get rid of the rest of it.